src/libs/deep-equal.ts

```diff
@@ -0,0 +1,31 @@
+const has = Object.prototype.hasOwnProperty
```

Comment:
Let's include https://github.com/lukeed/dequal/blob/master/license as a comment at the top of this file.

Reply:
The license must be included in the repo root and/or readme such that it remains visible in the npm package, as npm packages are copies ([substantial portions](https://github.com/lukeed/dequal/blob/9aa73181ac7e081cd330cac67d313632ac04bb02/license#L13)) of software.